docs/setup/arch/systems/README.md

```diff
@@ -0,0 +1,102 @@
+# FirecREST command execution
+
+## The simplistic approach
+
+The goal of FirecREST is to provide an HTTP interface to HPC, this means that the interface FirecREST presents to the users and applications must handle HTTP requests with according parameters, verbs, etc.
+
+FirecREST receives these requests and translates them into HPC logic, and after the operations are done (ie, commands executed, data uploaded, etc) it returns an HTTP response that complies with the protocol used.
+
+![f7t_simple](../../../assets/img/command_exec_simple.svg)
+
+## Going deeper
+
+The command execution from FirecREST API server to the HPC cluster is done **via SSH connection using HPC system users' credentials**.
+
+This means that FirecREST can be installed in any platform or infrastructure (cloud provider, VM, local system) with SSH access to the target system(s) configured. It doesn't need to be installed in the same VPN or VLAN of the system.
+
+> [!IMPORTANT]
+> FirecREST doesn't execute commands as `root` user or using `sudo` commands. All commands are executed on behalf of the users using the users' credential.
+
+## JWT to SSH Delegation
+
+To answer to the question on "how the SSH credentials are obtained or created", FirecREST provides an integration with JSON Web Tokens used to authenticate agains the API.
+
+FirecREST uses the `username` or `preferred_username` claim from the JWT access token created by the Identity Provider when the user or application [authenticated to use the API](../auth/README.md).
+
+> [!IMPORTANT]
+> This `username` value on the token must be the same username (`UID`) of the user on the HPC system. Otherwise the SSH credential created on its behalf won't be allowed in the system
```

Comment:
Maybe:
This `username` value on the token must be a valid username on the HPC system, otherwise the SSH credential created on its behalf won't be allowed in the system.